lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -348,6 +348,7 @@ def destroy(force_delete: false)
           }
           raise ActiveRecord::RecordInvalid unless @destroyed
 
+          @_swapped_id = duplicated_instance.swapped_id
```

Comment:
Is the swapped_id be assigned when `after_destroy` is called?
Intuitively, `after_destroy` is called after `_run_destroy_callback`, it seems that the swapped_id is not assigned at that time.